examples/backtrace_crate/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "backtrace_crate"
+version = "0.1.0"
+authors = ["Vardhan Thigle <vardhan.thigle@fortanix.com>"]
+edition = "2018"
+
+[dependencies]
+# backtrace = { path = "/home/vardhan/Work/GIT_REPOS/external/backtrace-rs", default-features = false,  features = ["std", "libunwind"]}
+backtrace = { path = "/home/vardhan/Work/GIT_REPOS/external/backtrace-rs"}
+# backtrace = { path = "/home/vardhan/Work/GIT_REPOS/external/backtrace-rs"}
```

Comment:
Fix this

Reply:
Thanks Jethro, will remove this entire example, as it is redundant.